### PR TITLE
Use 1e-6 as default tolerance value

### PR DESCRIPTION
**Pull request by lochel**
It seems that `OMEdit` and `omc` are using different tolerances for simulation by default.

@adeas31 Can you please review this changes? I try to fix lochel/PNlib#26.

### CODE REVIEW

**Comment by adeas31** [comment]
I let @sjoelund review it. I think he did this for some reason and I not sure why.

**Comment by sjoelund** [comment]
Ask @casella and @MartinOtter. Different tolerance settings are good for different domains. In OMEdit, I think a good default is 1e-4 in order to make models simulate faster. The user should anyway know when to tweak the tolerance setting. And for lochel/PNlib#26 a good fix (for all tools) would be to set a smaller tolerance in the experiment annotation.

**Comment by lochel** [comment]
Well, I see that there are good reasons for smaller/bigger tolerances. Shouldn’t be the default tolerance of `omc` and `OMEdit` the same anyway? And if `1e-4` is the better default value, then we should probably change the default tolerance of omc.

**Comment by casella** [comment]
Historically, 1e-4 has always been the default tolerance of Dymola. It is arguably tight enough for most applications, particularly the ones dealing with mechanical systems. In fact Simulink has an even looser default tolerance of 1e-3, which, incidentally, leads to results close to garbage status in many cases.
However, when comparing results obtained with Dymola and OMC, I can't remember exactly in which context (MSL or the AVM project), there were some significant mismatches, and it turned out that if the tolerance in Dymola was tightened to 1e-6, then the results would be coincident. Which basically meant, 1e-4 was not enough in some cases, and the results obtained with that tolerance were wrong. So, we decided that we put a premium on accuracy rather than performance, and left 1e-6 as default in OMC.

I am personally convinced that 1e-6 is a better (=safer) choice than 1e-4 as a default, and that obviously it should be the same in OMC and OMEdit. In those cases where 1e-4 can be proven a posteriori to be tight enough (= same results are obtained), or can avoid numerical problems, then one can add the experiment(Tolerance) annotation,  which is standardized, to the model.

Summing up, I am in favour of 1e-6 as default tolerance everywhere in the OpenModelica suite (OMC, OMEdit, OMNotebook, OMShell, etc.). If some user library has problems with that and requires a specific tolerance to work better, please set it using the experiment(Tolerance) annotation in the models.
